frame/system/src/lib.rs

```diff
@@ -200,12 +201,52 @@ pub mod pallet {
 	use crate::{self as frame_system, pallet_prelude::*, *};
 	use frame_support::pallet_prelude::*;
 
+	pub mod preludes {
+		use super::*;
+		pub mod testing {
+			type AccountId = u64;
+			use sp_runtime::traits::IdentityLookup;
+			use super::*;
+
+			pub struct Impl {}
+			impl DefaultConfig for Impl {
+				type Version = ();
+				type BlockWeights = ();
+				type BlockLength = ();
+				type DbWeight = ();
+				type Index = u64;
+				type BlockNumber = u64;
+				type Hash = sp_core::hash::H256;
+				type Hashing = sp_runtime::traits::BlakeTwo256;
+				type AccountId = AccountId;
+				type Lookup = IdentityLookup<AccountId>;
+				type BlockHashCount = frame_support::traits::ConstU64<10>;
+				type AccountData = u32;
+				type OnNewAccount = ();
+				type OnKilledAccount = ();
+				type SystemWeightInfo = ();
+				type SS58Prefix = ();
+				type MaxConsumers = ConstU32<16>;
+			}
+		}
+	}
+
 	/// System configuration trait. Implemented by runtime.
 	#[pallet::config]
+	// TODO: #[pallet::default_config]
```

Comment:
```suggestion
	// FAIL-CI TODO: #[pallet::default_config]
```
FYI you can use the CI to remind you of TODOs https://github.com/paritytech/ci_cd/issues/719 :smile:

Reply:
niceee